pkgs/tools/video/atomicparsley/default.nix

```diff
@@ -1,39 +1,29 @@
-{ stdenv, fetchhg, autoreconfHook, zlib, Cocoa }:
+{ stdenv, fetchFromGitHub, cmake, zlib, Cocoa }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "atomicparsley";
-  version = "0.9.6";
+  version = "20200701.154658.b0d6223";
 
-  src = fetchhg {
-    url = "https://bitbucket.org/wez/atomicparsley";
-    sha256 = "05n4kbn91ps52h3wi1qb2jwygjsc01qzx4lgkv5mvwl5i49rj8fm";
+  src = fetchFromGitHub {
+    owner = "wez";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-EHO4WkxoAXUhuJKMNYmBbGfOgtO9uklzXtWS4QsV1c8=";
   };
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ cmake ];
 
   buildInputs = [ zlib ]
-    ++ stdenv.lib.optionals stdenv.isDarwin [ Cocoa ];
+                ++ stdenv.lib.optionals stdenv.isDarwin [ Cocoa ];
 
-  configureFlags = stdenv.lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-    # AC_FUNC_MALLOC is broken on cross builds.
-    "ac_cv_func_malloc_0_nonnull=yes"
-    "ac_cv_func_realloc_0_nonnull=yes"
-  ];
-
-  installPhase = "install -D AtomicParsley $out/bin/AtomicParsley";
+  installPhase = ''
+    install -D AtomicParsley $out/bin/AtomicParsley
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`.

Reply:
Is this some new directive? I was not aware of it until now...